### ISSUE TITLE
Doesn't include public assets in nuxt v3.8.2.

### ISSUE THREAD

**Issue by RifatMahmudno-1** [vite-pwa/nuxt#94]
```js
workbox: {
	globPatterns: ['**/*.{js,css,html,png,svg,jpg,ttf}'],
	navigateFallback: '/?csr=1',
	cleanupOutdatedCaches: true
},
```
Nuxt version 3.8.2
@vite-pwa/nuxt 0.2.3

**Comment by userquin** [comment]
can you check if Rollup 4 or vite-plugin-pwa 0.17.0 present in your dependencies?

**Comment by RifatMahmudno-1** [comment]
I don't have Rollup or vite-plugin-pwa present in my dependencies.
Worked fine with older version of Nuxt (v3.7.4).

**Comment by userquin** [comment]
@RifatMahmudno-1 what command are you using build or generate?

**Comment by userquin** [comment]
> `npm run build` which runs `nuxt build`
thx, should be fixed in next minor `0.3.0`
